contentcuration/contentcuration/viewsets/task.py

```diff
@@ -26,7 +29,8 @@ class Meta:
 
 
 class TaskViewSet(ReadOnlyValuesViewset, DestroyModelMixin):
-    queryset = Task.objects.all()
+    order_by = 'created'
+    queryset = Task.objects.order_by(order_by)
```

Comment:
May be a better way to do this with the viewset?

Reply:
Could also just add it as the default ordering on the Task model?